packages/xarc-app-dev/src/lib/utils.ts

```diff
@@ -90,27 +91,36 @@ export const jsonParser = (key, value) => {
 
 let cachedXarcOptions;
 
-export function loadXarcOptions(dir: string = process.cwd()) {
+export function loadXarcOptions(dir: string = process.cwd(), showError = true) {
+  let cwd;
+  if (process.env.XARC_CWD) {
+    cwd = process.env.XARC_CWD;
+  } else {
+    cwd = dir;
+  }
   if (cachedXarcOptions) {
     return cachedXarcOptions;
   }
-  const filename = Path.join(dir, ".etmp/xarc-options.json");
+  const filename = Path.join(cwd, ".etmp/xarc-options.json");
   try {
     const data = Fs.readFileSync(filename, "utf-8");
     return (cachedXarcOptions = JSON.parse(data, jsonParser));
   } catch (err) {
     // eslint-disable-next-line
-    console.error(ck`
-<red>ERROR</>: Electrode xarc fail to load <cyan>.etmp/xarc-options.json</> in
-dev mode.  This means you are trying to use something not through
-xarc's development tasks.
-
-full path: ${filename}
+    if (showError) {
+      console.error(ck`
+      <red>ERROR</>: Electrode xarc fail to load <cyan>.etmp/xarc-options.json</> in
+      dev mode.  This means you are trying to use something not through
+      xarc's development tasks.
+      
+      full path: ${filename}
+      
+      Please run "clap setup-dev" once to initialize the file
+      <cyan>.etmp/xarc-options.json</> before doing your thing that loads
+      xarc's development code.
+      `);
```

Comment:
these extra spaces you added at the beginning of the lines will show in the actual output.